packages/logos-docusaurus-theme/src/client/components/mdx/Grid/Grid.tsx

```diff
@@ -14,10 +14,16 @@ import { GridItem } from './GridItem'
 
 export type GridProps = React.ComponentProps<typeof GridRoot> & {
   actions?: React.ReactNode
+  leftLabel?: string
+  rightLabel?: string
+  spaceBetween?: boolean
```

Comment:
`spaceBetween` could mean different things as a property of a grid component, but in this case, it only affects the scroll buttons. Please use a more clear name.